pkg/openthread/Kconfig

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2023 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_OPENTHREAD
```

Comment:
Given dependent modules, I think it'd be nicer to make this a `menuconfig`.

Reply:
> Given dependent modules, I think it'd be nicer to make this a `menuconfig`.

Thanks for the review and done that. I looks much nicer in `make menuconfig`